### PR TITLE
Added relevant links in emails to VG wiki/ onnline-offline mode in co…

### PR DESCRIPTION
**Pull request by borzamircea**
…nfig.properties file

### CODE REVIEW

File: verigreen-collector-impl/src/main/java/com/verigreen/collector/common/EmailSender.java [hunk 1]
```diff
@@ -140,6 +150,43 @@ public void notifyCommiter(
         if(subject.equals("Verigreen Status - Failure"))
         {
         	status.append(String.format("<tr><td>Temporary branch:</td><td>%s</td></tr></table>", tempBranch));
+        	if(offlineMode.equals("false"))
```
Comment (eitansch) [inline verigreen-collector-impl/src/main/java/com/verigreen/collector/common/EmailSender.java:156]: @borzamircea - I suggest the following:
1. Manage the URLs in a text file (ini, xml) and use generic tokens (GIT_FAILURE=q-i-got-a-git_failure-status-what-does-it-mean). This way, if we change the links, the logic remains the same (and we need to change the links in a text file and not in the code).
2. From code perspective, please do only string link = String.format("") inside the if's and in the end call one status.append(link) command at the bottom of the if (and else)

File: verigreen-collector-impl/src/main/java/com/verigreen/collector/common/EmailSender.java [hunk 2]
```diff
@@ -48,7 +57,7 @@ public String getCommitMessage(String commitId){
     	String commitMessage = null;
 		try {
 			FileRepositoryBuilder builder = new FileRepositoryBuilder();
-			String repoPath = VerigreenNeededLogic.properties.getProperty("git.repsoritoryLocation");
+			String repoPath = VerigreenNeededLogic.properties.getProperty("git.repositoryLocation");
```
Comment (eitansch) [inline verigreen-collector-impl/src/main/java/com/verigreen/collector/common/EmailSender.java:61]: More spelling mistakes? Have you checked on both Win/Lin it still works?

**Comment by eitansch** [comment]
@borzamircea - where are the changes to the config.properties file?

**Comment by borzamircea** [comment]
There should be a line a the end:

#Enable offline FAQ page
faq.offline=false

**Comment by eitanshp** [comment]
Why don't I see the changes in the commit for the config.properties file?

**Comment by soninob** [comment]
@eitanshp - are we agreeing on the offline mode (adding the FAQ.html to our code repository)?

**Comment by eitansch** [comment]
The offline mode was suggested by me and implemented by @borzamircea.
Please merge it to the master.

**Comment by borzamircea** [comment]
Closed by #71

**Pull request closed without merging**